src/modules/UI/components/Buttons/TertiaryButton.ui.js

```diff
@@ -30,7 +30,7 @@ export class TertiaryButton extends React.Component<Props> {
   render() {
     const { children, style, ...props } = this.props
     return (
-      <TouchableHighlight underlayColor={rawStyles.tertiaryButtonUnderlay.color} style={[styles.button, styles.tertiaryButton, style]} {...props}>
+      <TouchableHighlight underlayColor={tertiaryButtonUnderlay.color} style={[styles.button, styles.tertiaryButton, style]} {...props}>
```

Comment:
does this work? you're passing a style parameter into a non-style prop. The previous code looks more correct

Reply:
The `underlayColor` is a [normal prop](https://reactnative.dev/docs/0.68/touchablehighlight#underlaycolor), not a style for some reason. So we need to pass it as a normal value, which is really weird but correct.